addons/l10n_pe/data/account_tax_data.xml

```diff
@@ -1,56 +1,84 @@
 <?xml version="1.0" encoding="utf-8"?>
 <odoo>
+    <record id="tax_group_02" model="account.tax.group">
+        <field name="name">IGV 2%</field>
+    </record>
+    <record id="tax_group_18" model="account.tax.group">
+        <field name="name">IGV 18%</field>
+    </record>
+    <record id="tax_tag_sale_02" model="account.account.tag">
+        <field name="name">IGV 2% venta</field>
+        <field name="applicability">taxes</field>
+    </record>
+
+    <record id="tax_tag_purchase_02" model="account.account.tag">
+        <field name="name">IGV 2% compras</field>
+        <field name="applicability">taxes</field>
+    </record>
+
     <record id="tax_tag_sale_18" model="account.account.tag">
         <field name="name">IGV 18% Venta</field>
         <field name="applicability">taxes</field>
     </record>
 
     <record id="tax_tag_purchase_18" model="account.account.tag">
-        <field name="name">IGV 18% Compra</field>
+        <field name="name">IGV 18% compra</field>
         <field name="applicability">taxes</field>
     </record>
 
-    <record id="tax_tag_purchase_02" model="account.account.tag">
-        <field name="name">Percepción IGV 2%</field>
-        <field name="applicability">taxes</field>
+    <record id="sale_tax_02" model="account.tax.template">
+      <field name="chart_template_id" ref="pcge"/>
+      <field name="name">IGV 2% Venta</field>
+      <field name="description">IGV 2% Venta</field>
+      <field name="amount">2</field>
+      <field name="amount_type">percent</field>
+      <field name="type_tax_use">sale</field>
+      <field name="sequence">2</field>
+      <field name="account_id" ref="acc401111"/>
+      <field name="refund_account_id" ref="acc401111"/>
+      <field name="tag_ids" eval="[(6,0,[ref('tax_tag_sale_02')])]"/>
+      <field name="tax_group_id" ref="tax_group_02"/>
     </record>
 
-    <record id="ITAX_18" model="account.tax.template">
-      <field name="chart_template_id" ref="pe_chart_template"/>
+    <record id="sale_tax_18" model="account.tax.template">
+      <field name="chart_template_id" ref="pcge"/>
       <field name="name">IGV 18% Venta</field>
       <field name="description">IGV 18% Venta</field>
       <field name="amount">18</field>
       <field name="amount_type">percent</field>
       <field name="type_tax_use">sale</field>
-      <field name="account_id" ref="401110"/>
-      <field name="refund_account_id" ref="401110"/>
+      <field name="sequence">1</field>
+      <field name="account_id" ref="acc401111"/>
+      <field name="refund_account_id" ref="acc401111"/>
       <field name="tag_ids" eval="[(6,0,[ref('tax_tag_sale_18')])]"/>
       <field name="tax_group_id" ref="tax_group_18"/>
     </record>
 
-    <record id="OTAX_18" model="account.tax.template">
-      <field name="chart_template_id" ref="pe_chart_template"/>
+    <record id="vendor_tax_02" model="account.tax.template">
```

Comment:
I was thinking about keeping the same xml_id here, but I don't think it will improve anything, because the tags itself are kept.  (and the new ones are for new taxes)  Btw, do we have an idea what to do with these tags afterwards?

Reply:
This tags will be used in l10n_pe_reports (once done) to set the proper accounting reports, for now on, they are just basic data.